GetCertificate/printer.ts

```diff
@@ -179,7 +181,16 @@ export const printUvci = (
  * @returns true if process ended, false otherwise
  */
 export const isVaccinationProcessEnded = (v: VaccinationEntry): boolean =>
-  v.dn === v.sd;
+  v.dn >= v.sd;
+
+/**
+ * Test validity in hours, based on type
+ *
+ * @param v the VaccinationEntry to check
+ * @returns the validity of the test, in hours
+ */
+export const testValidity = (v: TestEntry): string =>
```

Comment:
Just a hint: we have tagged types that may help enforcing functions signature when dealing with time units: https://github.com/pagopa/io-ts-commons/blob/e107a4308d9d81fb2c944db364c15b56ff5792c8/src/units.ts#L15

Reply:
Thanks, interesting. Do you think we should move to tagged type in this case?